tensorflow_quantum/datasets/spin_system.py

```diff
@@ -320,3 +320,250 @@ def tfi_chain(qubits, boundary_condition="closed", data_dir=None):
             additional_info)))
 
     return resolved_circuits, labels, hamiltonians, additional_info
+
+
+def xxz_chain(qubits, boundary_condition="closed", data_dir=None):
+    """XXZ model quantum data set.
+
+    $$
+    H = \sum_{i} \sigma_i^x \sigma_{i+1}^x + \sigma_i^y \sigma_{i+1}^y +
+    \Delta\sigma_i^z \sigma_{i+1}^z
+    $$
+
+    Contains 76 circuit parameterizations corresponding to
+    the ground states of the 1D XXZ chain for g in [0.3,1.8].
+    This dataset contains 76 datapoints. Each datapoint is represented by a
+    circuit (`cirq.Circuit`), a label (Python `float`) a Hamiltonian
+    (`cirq.PauliSum`) and some additional metadata. Each Hamiltonian in a
+    datapoint is a 1D XXZ chain with boundary condition `boundary_condition` on
+    `qubits` whos order parameter dictates the value of label. The circuit in a
+    datapoint prepares (an approximation to) the ground state of the Hamiltonian
+    in the datapoint.
+
+    Example usage:
```

Comment:
Nit: space.

Reply:
I feel dumb for asking this, but what do you mean by this? Add/remove a space?

Reply:
Oh.... I weirdly thought you had an extra space in front of the E indenting it farther than everything else. Looking again it's toally fine hehe.

Reply:
Haha I see.